src/UI.js

```diff
@@ -0,0 +1,178 @@
+import TaskList from './TaskList.js';
+import deleteIco from './delete.svg';
+import enterIco from './return.svg';
+import moreIco from './more.svg';
+import renewIco from './renew.svg';
+
+class UI {
+  static #heading = document.querySelector('.heading');
+
+  static #newItem = document.querySelector('.input-row');
+
+  static #taskNew = document.querySelector('#task-new');
+
+  static #taskList = document.querySelector('#todo-list');
+
+  static #template = document.querySelector('li');
+
+  static #clearBtn = document.querySelector('button');
+
+  static getCheckBox = (taskElement) => taskElement.querySelector('input[type="checkbox"]');
+
+  static getDeleBtn = (taskElement) => taskElement.querySelector('#dele');
+
+  static getMoreBtn = (taskElement) => taskElement.querySelector('#more');
+
+  static getTaskEdit = (taskElement) => taskElement.querySelector('#task-edit');
+
+  static setFormIcons = () => {
+    const renewIcon = new Image();
```

Comment:
For best javascript practice, it is advised not to use a new object instance because it is a primitive way of creating a javaScript object or attribute.

Reply:
@idungstanley 
I agree with you on the fact that it is advised no to use a new Object, **when there's a better way to create it, using a constant**.

Nevertheless, in this specific case, there are only 2 alternatives way to create a DOM Element, and I chose the first one.

1. const renewIcon = new Image();
2. const renewIcon = document.createNewElement('img');

I hope you take this clarification into consideration to receive your approval.

Here is an screenshot of the advise from [https://www.w3schools.com/js/js_best_practices.asp](https://www.w3schools.com/js/js_best_practices.asp)
![image](https://user-images.githubusercontent.com/100373071/166330350-9999c72b-ba5a-490b-8aa3-2d185ffa9884.png)